test/plots/piePlotTests.ts

```diff
@@ -427,113 +406,113 @@ describe("Plots", () => {
-        let entity3 = piePlot.entitiesAt(click3);
-        assert.strictEqual(entity3.length, 0, "no entities returned");
+      it("returns nothing for points within innerRadius() or outside of outerRadius() with entitiesAt()", () => {
+        let INNER_RADIUS = 100;
+        piePlot.innerRadius(INNER_RADIUS);
+
+        let clickInCenter = {
+          x: 200,
+          y: 200
+        };
+        let entitiesInCenter = piePlot.entitiesAt(clickInCenter);
+        assert.lengthOf(entitiesInCenter, 0, "no entities returned when clicking inside innerRadius()");
+
+        let OUTER_RADIUS = 150;
+        piePlot.outerRadius(OUTER_RADIUS);
+        let clickBetweenRadii = {
+          x: 200,
+          y: 200 + (INNER_RADIUS + OUTER_RADIUS) / 2
+        };
+        let entitiesBetweenRadii = piePlot.entitiesAt(clickBetweenRadii);
+        TestMethods.assertPlotEntitiesEqual(entitiesBetweenRadii[0], piePlot.entities()[1], "retrieved the correct entity");
+
+        let clickOutsideOuterRadius = {
+          x: 200,
+          y: 200 + OUTER_RADIUS + 1
+        };
+        let entitiesOutsideOuterRadius = piePlot.entitiesAt(clickOutsideOuterRadius);
+        assert.lengthOf(entitiesOutsideOuterRadius, 0, "no entities returned when clicking outside outerRadius()");
         svg.remove();
       });
-
     });
 
-    describe("Fill", () => {
-
-      it("sectors are filled in according to defaults", () => {
-        let arcPaths = renderArea.selectAll(".arc");
-
-        let arcPath0 = d3.select(arcPaths[0][0]);
-        assert.strictEqual(arcPath0.attr("fill"), "#5279c7", "first sector filled appropriately");
-
-        let arcPath1 = d3.select(arcPaths[0][1]);
-        assert.strictEqual(arcPath1.attr("fill"), "#fd373e", "second sector filled appropriately");
+    describe("Fail safe tests", () => {
+      it("renders correctly with no Datasets", () => {
+        let SVG_WIDTH = 400;
+        let SVG_HEIGHT = 400;
+        let svg = TestMethods.generateSVG(SVG_WIDTH, SVG_HEIGHT);
+        let plot = new Plottable.Plots.Pie();
+        plot.sectorValue((d) => d.value);
+        assert.doesNotThrow(() => plot.renderTo(svg), Error);
+        assert.strictEqual(plot.width(), SVG_WIDTH, "was allocated width");
+        assert.strictEqual(plot.height(), SVG_HEIGHT, "was allocated height");
         svg.remove();
       });
 
-      it("project fill", () => {
-        piePlot.attr("fill", (d: any, i: number) => String(i), new Plottable.Scales.Color("10"));
+      it("does not render slices for undefined, NaN, strings, or negative numbers", () => {
+        let svg = TestMethods.generateSVG();
+
+        let dataWithBadValue = [
+          { v: 1 },
+          { v: undefined },
+          { v: 1 },
+          { v: NaN },
+          { v: 1 },
+          { v: "Bad String" },
+          { v: 1 },
+          { v: -100 }
+        ];
 
-        let arcPaths = renderArea.selectAll(".arc");
+        let plot = new Plottable.Plots.Pie();
```

Comment:
Let's standardize the name of the plot. I am okay with either `plot` or `piePlot`, but once we decide let's add it to the testing standards to be consistent within file and across files.